### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by fao89**
CI: https://github.com/openshift/release/pull/35732

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -0,0 +1,19 @@
+approvers:
+  - abays
+  - dprince
+  - rebtoor
+  - olliewalsh
+  - Sandeepyadav93
+  - stuggi
+  - viroel
+  - fao89
```
Comment (slagle) [inline OWNERS:9]: add slagle

File: OWNERS [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+reviewers:
+  - abays
+  - dprince
+  - rebtoor
+  - olliewalsh
+  - Sandeepyadav93
+  - stuggi
+  - viroel
+  - fao89
```
Comment (slagle) [inline OWNERS:20]: add slagle

File: Makefile [hunk 3]
```diff
@@ -86,8 +88,12 @@ help: ## Display this help.
 ##@ Development
 
 .PHONY: manifests
-manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+manifests: controller-gen crd-to-markdown ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
+	$(CONTROLLER_GEN) rbac:roleName=manager-role crd$(CRDDESC_OVERRIDE) webhook paths="./..." output:crd:artifacts:config=config/crd/bases && \
+	rm -f api/bases/* && cp -a config/crd/bases api/
```
Comment (slagle) [inline Makefile:93]: what's the purpose of this line? It just copies the CRD's into api/bases? Why is that needed?

Comment (fao89) [reply Makefile:93]: I just copied it from https://github.com/openstack-k8s-operators/openstack-ansibleee-operator/blob/main/Makefile#L94-L97

all the changes of the Makefile came from there

File: OWNERS (hunk 1 above)
Comment (fao89) [reply OWNERS:9]: done!

File: OWNERS (hunk 2 above)
Comment (fao89) [reply OWNERS:20]: done!

File: Makefile (hunk 3 above)
Comment (fao89) [reply Makefile]: I've removed it
